types/src/stored_value.rs

```diff
@@ -909,14 +934,38 @@ impl<'de> Deserialize<'de> for StoredValue {
 mod tests {
     use crate::{bytesrepr, gens, StoredValue};
     use proptest::proptest;
+    use serde_json::Value;
+    const STORED_VALUE_CONTRACT_PACKAGE_RAW: &str = "{\"ContractPackage\":{\"access_key\":\"uref-024d69e50a458f337817d3d11ba95bdbdd6258ba8f2dc980644c9efdbd64945d-007\",\"versions\":[{\"protocol_version_major\":1,\"contract_version\":1,\"contract_hash\":\"contract-1b301b49505ec5eaec1787686c54818bd60836b9301cce3f5c0237560e5a4bfd\"}],\"disabled_versions\":[],\"groups\":[],\"lock_status\":\"Unlocked\"}}";
```

Comment:
We could use a raw string literal here for increased readability:
```suggestion
const STORED_VALUE_CONTRACT_PACKAGE_RAW: &str = r#"
        {
            "ContractPackage":
            {
                "access_key":"uref-024d69e50a458f337817d3d11ba95bdbdd6258ba8f2dc980644c9efdbd64945d-007",
                "versions":
                [
                    {
                        "protocol_version_major":1,
                        "contract_version":1,
                        "contract_hash":"contract-1b301b49505ec5eaec1787686c54818bd60836b9301cce3f5c0237560e5a4bfd"
                    }
                ],
                "disabled_versions":[],
                "groups":[],
                "lock_status":"Unlocked"
            }
        }"#;
```